src/AzureRenderManager/Microsoft.Identity.Web/Client/ITokenAcquisition.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.AspNetCore.Authentication.OpenIdConnect;
```

Comment:
Do we need to add additional license comments or details about where this code came from?

Reply:
Some have the appropriate headers, some don't.  All MIT.  I've added a missing header and comment about source.  Not ideal, better than nothing I guess though.